modules/travel_pay/app/models/travel_pay/base_expense.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class BaseExpense
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+    include ActiveModel::Validations
+
+    attribute :purchase_date, :datetime
+    attribute :description, :string
+    attribute :cost_requested, :float
```

Comment:
@seesleestak Is there a specific reason you want the `requested_cost` to be a float? Recommendation: `attribute :cost_requested, :decimal, precision: 8, scale: 2`

Reply:
@kjduensing Rounding errors, but it is only a recommendation. You know your use case best.

Reply:
If it's okay, I'd like to stick with the float type here. I don't believe our use case will be hurt by that choice.